--- wayfire-0.8.0/meson.build.orig	2024-06-28 23:37:15.218293323 +0200
+++ wayfire-0.8.0/meson.build	2024-06-29 00:00:36.554167930 +0200
@@ -25,7 +25,10 @@
 egl            = dependency('egl')
 glesv2         = dependency('glesv2')
 vulkan         = dependency('vulkan', required: false)
+glm = dependency('glm', modules: ['glm::glm-header-only'], required: false)
+if not glm.found()
 glm            = dependency('glm', required: false)
+endif
 libinput       = dependency('libinput', version: '>=1.7.0')
 pixman         = dependency('pixman-1')
 xkbcommon      = dependency('xkbcommon')
--- wayfire-0.9.0/subprojects/wf-touch/meson.build.orig	2024-04-01 14:15:12.000000000 +0200
+++ wayfire-0.9.0/subprojects/wf-touch/meson.build	2024-10-11 13:19:24.416967488 +0200
@@ -3,9 +3,12 @@
 		    meson_version: '>=0.47.0',
 		    default_options : ['cpp_std=c++17'])
 
-glm = dependency('glm', required: false)
-if not glm.found() and not meson.get_compiler('cpp').check_header('glm/glm.hpp')
-  error('GLM not found, and directly using the header \'glm/glm.hpp\' is not possible.')
+glm = dependency('glm', modules: ['glm::glm-header-only'], required: false)
+if not glm.found()
+  glm = dependency('glm', required: false)
+  if not glm.found() and not meson.get_compiler('cpp').check_header('glm/glm.hpp')
+    error('GLM not found, and directly using the header \'glm/glm.hpp\' is not possible.')
+  endif
 endif
 
 wf_touch_inc_dirs = include_directories('.')
